test/integration/testdata/dummytype/alpha.go

```diff
@@ -0,0 +1,27 @@
+package dummytype
+
+// The package is intended for testing the openapi-gen API rule
+// checker. The API rule violations are in format of:
+//
+// `{rule-name},{package},{type},{(optional) field}`
+//
+// The checker should sort the violations before
+// reporting to a file or stderr.
+
+// +k8s:openapi-gen=true
+type Foo struct {
+	B string
```

Comment:
I find the letters a little bit confusing, maybe it's just me

Reply:
I agree. Especially the `CD` can be confusing. I can add some comments in the type def to document that the letters are just for testing the behavior of sorting the parameters relatively and do not take any real meaning. Do you have any other suggestions?

Reply:
I'd just come-up with random names